apps/react/react-2-native.json

```diff
@@ -89,7 +69,7 @@
     {
       "label": "cli help",
       "description": "Click here to know more cli configurations",
-      "command": "npx expo init --help"
+      "command": "npx expo --help"
```

Comment:
Please change the command to `npx create-expo-app --help`

Reply:
done